### PR TITLE
SYS-129 github migration

### PR DESCRIPTION
**Pull request by chrypnotoad**
https://linear.app/shm/issue/SYS-129/point-dashboard-installer-to-github-repos

Summary: changing gitlab references to github

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -24,14 +24,14 @@ echo "/home/node/.pm2/logs/*.log /home/node/app/cli/build/logs/*.log {
 
 # Pull latest versions of the CLI and GUI
 
-git clone https://gitlab.com/shardeum/validator/cli.git
+git clone -b sys-129-point-dashboard-installer-to-github-repos https://github.com/shardeum/validator-cli.git cli
```
Comment (muni-corn) [inline entrypoint.sh:27]: Will we want to remove this `-b` argument before merging?

**Review by muni-corn — DISMISSED** [review]
Approving, just want to clear the `-b` argument being passed to git commands

File: entrypoint.sh (hunk 1 above)
Comment (chrypnotoad) [reply entrypoint.sh:27]: yes! this was so QA could test the install

**Review by muni-corn — APPROVED** [review]
Interesting! Looks good to me!

**Pull request merged**